<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PurchaseDialog</class>
 <widget class="QDialog" name="PurchaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(193, 207, 161);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>226</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 8, 12);</string>
   </property>
   <property name="text">
    <string>PURCHASE BOOK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>461</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 8, 12);</string>
    </property>
    <property name="text">
     <string>Book Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 8, 12);</string>
    </property>
    <property name="text">
     <string>Quantity:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Purchase_BookName_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color: rgb(255, 0, 77);
	background-color: rgb(198, 201, 255);
	border: 3px solid #ff2bed;
	border-radius: 7px;

}</string>
    </property>
    <property name="placeholderText">
     <string>Enter book name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BookQuantity_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color: rgb(255, 0, 77);
	background-color: rgb(198, 201, 255);
	border: 3px solid #ff2bed;
	border-radius: 7px;

}</string>
    </property>
    <property name="placeholderText">
     <string>Enter book quantity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Purchase_Book_Button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(198, 201, 255);
color: rgb(38, 173, 14);
border-radius: 10px;
border: 3px solid #5c7fff;
}

QPushButton:hover{
	background-color: rgb(101, 134, 255);
	color: rgb(255, 123, 215)
border-radius: 10px;
border:5px solid #ff582e;

}

QPushButton:pressed{
	background-color: rgb(125, 255, 227);
	background-color: rgb(255, 123, 174);
border-radius: 10px;
border: 5px solid  rgb(0, 0, 127);
}
</string>
    </property>
    <property name="text">
     <string>PURCHASE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
